src/qililab/instruments/awg_analog_digital_converter.py

```diff
@@ -330,6 +333,19 @@ def _set_scope_store_enabled(self, value: float | str | bool, sequencer_id: int)
         """
         cast(AWGADCSequencer, self.get_sequencer(sequencer_id)).scope_store_enabled = bool(value)
 
+    @Instrument.CheckParameterValueFloatOrInt
+    def _set_time_of_flight(self, value: float | str | bool, sequencer_id: int):
+        """set scope_store_enable
```

Comment:
This docstring is fir the scope_store enable.
Also the value may also be a Bool, is that right? Shouldn't it be an int (float at best if the user sets an int as a float)?

Reply:
fixed